.github/workflows/ci.yml

```
@@ -83,7 +83,7 @@ jobs:
           python -m pip install pytest pytest-pretty scikit-image[data] zarr xarray hypothesis matplotlib
 
       - name: Run napari plugin headless tests
-        run: pytest -W 'ignore::DeprecationWarning' napari/plugins napari/settings napari/layers napari/components
+        run: pytest -W 'ignore::DeprecationWarning' napari/src/plugins napari/src/settings napari/src/layers napari/src/components
```

```suggestion
        run: pytest -W 'ignore::DeprecationWarning' src/napari/plugins src/napari/settings src/napari/layers src/napari/components
```
I think you meant this way? since the code is under napari in the src dir?

---

Shhh you didn't see anything :dotted_line_face: